core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -378,38 +381,20 @@ public static Function createCursorFunction(
         return function;
     }
 
-    public static void createMatViewDefinitionFile(MemoryMARW mem, MatViewDefinition matViewDefinition) {
-        mem.extend(MV_HEADER_SIZE);
-        mem.jumpTo(MV_HEADER_SIZE);
+    public static void createMatViewDefinition(AppendableBlock mem, MatViewDefinition matViewDefinition) {
```

Comment:
It might be a better idea to accept `MetaFileWriter` here since we always want to append and then commit.